packages/odyssey-react-mui/src/labs/SideNav/SideNavItemLinkContent.tsx

```diff
@@ -62,11 +66,13 @@ const SideNavItemLinkContent = ({
         odysseyDesignTokens={odysseyDesignTokens}
         isIconVisible={Boolean(startIcon)}
       >
-        {Boolean(startIcon)}
         {label}
-        {severity && (
+        {(severity || count) && (
           <Box sx={sideNavItemContentStyles}>
-            <Status severity={severity} label={statusLabel || ""} />
+            {count && <Badge badgeContent={count} />}
+            {severity && (
+              <Status severity={severity} label={statusLabel || ""} />
+            )}
```

Comment:
Is it possible to have both severity and count? I would make them mutually independent just in case until we have more concrete requirements.